src/routes/index.js

```diff
@@ -23,7 +23,13 @@ export default function getRoutes(renderRequestMap, apiProxyFn$ = apiProxy$) {
 		method: 'GET',
 		path: '/_ah/{param*}',
 		config: { auth: false },
-		handler: (request, reply) => reply('OK'),
+		handler: (request, reply) => {
+			if (request.params.param == !'error') {
+				reply('OK');
+			} else {
+				throw new Error('Simulated error via url');
```

Comment:
A helpful convention is to handle errors at the top of your function and then keep the 'happy path' left-aligned - it reduces nesting and makes the error conditions a little easier to identify

```js
if (request.params.param == 'error') {
  throw new Error();
}
reply('OK');
```